extensions/v2ex/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "v2ex",
+  "title": "v2ex",
+  "description": "hot list of v2ex",
+  "icon": "icon-v2ex.png",
+  "author": "leo",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show v2ex List",
+      "subtitle": "Top 10 hot topics currently",
```

Comment:
I'd leave the subtitle empty and rather update the command title, for example: "Show v2ex Top 10"
Please also see our guidelines for title and subtitle properties: https://developers.raycast.com/basics/prepare-an-extension-for-store

Reply:
I have updated the title and subtitle, thanks.